### PR TITLE
Replace travis with Github Actions

### PR DESCRIPTION
**Pull request by Luni-4**
This PR replaces `travis` with `Github Actions`

Thanks in advance for your review! :)

### CODE REVIEW

File: .github/workflows/ebur128.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        profile: minimal
+        toolchain: stable
+        override: true
+        components: clippy, rustfmt
+
+    - name: Run rustfmt
+      uses: actions-rs/cargo@v1
+      with:
+        command: fmt
+        args: -- --check --verbose
```
Comment (sdroege) [inline .github/workflows/ebur128.yml]:
```suggestion
        args: -- --check
```

The additional output is not very useful and just internal stuff :)

File: .github/workflows/ebur128.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+    - name: Run clippy
+      uses: actions-rs/clippy-check@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        args: -- -D warnings --verbose -A clippy::wrong-self-convention
```
Comment (sdroege) [inline .github/workflows/ebur128.yml]:
```suggestion
        args: -- -D warnings -A clippy::wrong-self-convention
```

Same as above :) And probably want to add `--all-features --all-targets` or similar, and ignore (via `#[allow(missing_safety_doc)]`) for the `capi` module (and only that).

It looks also like `clippy::wrong-self-convention` doesn't have to be ignored? I don't have clippy complaining about that anywhere.

File: .github/workflows/ebur128.yml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+    - name: Check capi
+      run: |
+        make -C capi-test check
+
+    - name: Check capi valgrind
+      run: |
+        make -C capi-test check-valgrind
+
+  unix-tests:
```
Comment (sdroege) [inline .github/workflows/ebur128.yml]:
```suggestion
  ubuntu-latest-tests:
```

Or so maybe?

**Review by sdroege — APPROVED** [review]
Thanks, generally looks good to me (and also the run of it in your fork) :)

File: .github/workflows/ebur128.yml (hunk 2 above)
Comment (Luni-4) [reply .github/workflows/ebur128.yml]: I hope I've figured out correctly :)

Comment (sdroege) [reply .github/workflows/ebur128.yml]: Not exactly :) Should be one job/action/task (how is it called?) with clippy for all the targets/features, but you need to add an `#[allow(clippy::missing_safety_doc)]` in `src/lib.rs` before the `mod capi` line.

Comment (Luni-4) [reply .github/workflows/ebur128.yml]: Oh, in that sense, thanks!

Comment (Luni-4) [reply .github/workflows/ebur128.yml]: Done! :)

**Comment by sdroege** [comment]
Thanks!

**Pull request merged**